lib/bookingsync/api/client/bookings_comments.rb

```diff
@@ -0,0 +1,72 @@
+module BookingSync::API
+  class Client
+    module BookingsComments
+      # List bookings comments
+
+      # Returns bookings comments for the account and rentals user is authenticated with.
+      # @param options [Hash] A customizable set of options.
+      # @option options [Integer] rental_id: Rental ID that booking is assiociated with.
+      # @return [Array<BookingSync::API::Resource>] Array of comments.
+      #
+      # @example Get the list of comments for the current account
+      #   @api.bookings_comments
+      # => [{:links=>{:booking=>10}, :id=>1, :content=>"comment 1", :editable=>true, :created_at=>2016-04-18 12:06:15 UTC, :updated_at=>2016-04-18 12:06:15 UTC},
+      #     {:links=>{:booking=>11}, :id=>2, :content=>"comment 2", :editable=>true, :created_at=>2016-04-18 12:06:25 UTC, :updated_at=>2016-04-18 12:06:25 UTC}]
+      # @example Get the list of comments only for specific rental
```

Comment:
yeah, there is `for_rentals` scope, but can't really imagine filtering booking_comments by rental_id :P